packages/react-instantsearch-hooks-dom/src/widgets/__tests__/all-components.test.tsx

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
Do we need this test file? We're testing this in each individual widget already.

Reply:
this is making sure we never forget it in a test. See it as contract-testing